### PR TITLE
fix shadowing and other warnings

### PR DESCRIPTION
**Pull request by gandrewstone**

### CODE REVIEW

**Comment by ptschip** [comment]
needs a format

File: src/graphene_set.h [hunk 1]
```diff
@@ -297,7 +298,8 @@ class CGrapheneSet
         size_t nEncodedWords = int(ceil(nBitsPerItem * nItems / float(WORD_BITS)));
 
         // decode into boolean array (low-order first)
-        bool bits[nEncodedWords * WORD_BITS];
+        bool* bits = new bool[nEncodedWords * WORD_BITS];
+
```
Comment (ptschip) [inline src/graphene_set.h]: what about making this a unique_ptr?

File: src/net.cpp [hunk 2]
```diff
@@ -1600,6 +1600,7 @@ static void DNSAddressSeed()
     LOGA("%d addresses found from DNS seeds\n", found);
 }
 
+#if 0
 // BITCOINUNLIMITED START
```
Comment (ptschip) [inline src/net.cpp]: maybe a comment that this is temporarily disabled?

**Review by ptschip — APPROVED** [review]
looks good but travis had a spurious failure, you might want to run travis on more time.

**Review by sickpig — APPROVED** [review]

**Pull request merged**